glotaran/optimization/test/test_constraints.py

```diff
@@ -17,7 +17,7 @@ def test_constraint(index_dependent, link_clp):
     model.megacomplex["m1"].is_index_dependent = index_dependent
     model.clp_constraints.append(ZeroConstraint.from_dict({"target": "s2"}))
 
-    print("link_clp", link_clp, "index_dependent", index_dependent)
+    print("link_clp", link_clp, "index_dependent", index_dependent)  # T201
```

Comment:
This is an artifact of a badly written noqa and a using `yesqa` see [this commit](https://github.com/glotaran/pyglotaran/commit/3c21f46d8c7a6ecfe5058cc2cc094cef5dd55cf2)